iib/web/api_v1.py

```diff
@@ -557,6 +557,7 @@ def regenerate_bundle_batch():
                     build_request['from_bundle_image'],
                     build_request.get('organization'),
                     request.id,
+                    build_request.get('registry_auths'),
```

Comment:
We need to provide `argsrepr` parameter to the `apply_async` call because `registry_auths` contains sensitive information that should not be logged. Probably something like this:

```python
args = [
  build_request['from_bundle_image'],
  build_request.get('organization'),
  request.id,
  build_request.get('registry_auths'),
]
safe_args = _get_safe_args(args, build_request)
handle_regenerate_bundle_request.apply_async(
  args=args.
  argsrepr=safe_args,
  link_error=error_callback,
  queue=_get_user_queue(),
)
```

Be sure to add a test to make sure the `registry_auths` value is redacted in `argsrepr`. Please, also validate that the logs show the redacted value.

Reply:
+1